app/entities/sessions.entity.ts

```diff
@@ -1,6 +1,10 @@
 import { MinLength, IsNotEmpty } from 'class-validator'
 import {
-  Entity, BaseEntity, ObjectIdColumn, CreateDateColumn, UpdateDateColumn,
+  Entity,
+  BaseEntity,
+  ObjectIdColumn,
+  CreateDateColumn,
+  UpdateDateColumn
```

Comment:
```suggestion
  UpdateDateColumn,
```
The last line always has a comma, keep up with the old style.

Reply:
Alright, I'll put trailing commas into the Prettier config